mmdet/datasets/mmfashion.py

```diff
@@ -0,0 +1,10 @@
+from .builder import DATASETS
+from .coco import CocoDataset
+
+
+@DATASETS.register_module()
+class mmfashionDataset(CocoDataset):
```

Comment:
Rename it to `DeepFashionDataset`?

Reply:
Renamed mmfashion to DeepFashion in the new commit.